pylint/checkers/base.py

```diff
@@ -1377,7 +1377,7 @@ def visit_continue(self, node: nodes.Continue) -> None:
     def visit_break(self, node: nodes.Break) -> None:
         """1 - check is the node has a right sibling (if so, that's some
         unreachable code)
-        2 - check is the node is inside the finally clause of a try...finally
+        2 - check is the node is inside the 'finally' clause of a try...finally
```

Comment:
```suggestion
        2 - check if the node is inside the 'finally' clause of a try...finally
```

Reply:
You added quotes to other occurrences of `try...finally`. Do you want to do that here as well?

Reply:
Nice catch, it was not detected by my grammer linter :)
```suggestion
        2 - check if the node is inside the 'finally' clause of a 'try...finally'
```